diff --git a/build/fbcode_builder/CMake/FindGlog.cmake b/build/fbcode_builder/CMake/FindGlog.cmake
deleted file mode 100644
index a589b2e..0000000
--- a/build/fbcode_builder/CMake/FindGlog.cmake
+++ /dev/null
@@ -1,32 +0,0 @@
-# Copyright (c) Facebook, Inc. and its affiliates.
-# - Try to find Glog
-# Once done, this will define
-#
-# GLOG_FOUND - system has Glog
-# GLOG_INCLUDE_DIRS - the Glog include directories
-# GLOG_LIBRARIES - link these to use Glog
-
-include(FindPackageHandleStandardArgs)
-
-find_library(GLOG_LIBRARY glog
-  PATHS ${GLOG_LIBRARYDIR})
-
-find_path(GLOG_INCLUDE_DIR glog/logging.h
-  PATHS ${GLOG_INCLUDEDIR})
-
-find_package_handle_standard_args(glog DEFAULT_MSG
-  GLOG_LIBRARY
-  GLOG_INCLUDE_DIR)
-
-mark_as_advanced(
-  GLOG_LIBRARY
-  GLOG_INCLUDE_DIR)
-
-set(GLOG_LIBRARIES ${GLOG_LIBRARY})
-set(GLOG_INCLUDE_DIRS ${GLOG_INCLUDE_DIR})
-
-if (NOT TARGET glog::glog)
-  add_library(glog::glog UNKNOWN IMPORTED)
-  set_target_properties(glog::glog PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${GLOG_INCLUDE_DIRS}")
-  set_target_properties(glog::glog PROPERTIES IMPORTED_LINK_INTERFACE_LANGUAGES "C" IMPORTED_LOCATION "${GLOG_LIBRARIES}")
-endif()
diff --git a/fizz/CMakeLists.txt b/fizz/CMakeLists.txt
index f69fc31..16c6ab2 100644
--- a/fizz/CMakeLists.txt
+++ b/fizz/CMakeLists.txt
@@ -45,7 +45,7 @@ endif()
 find_package(Boost REQUIRED COMPONENTS system thread filesystem regex context
                                        program_options)
 find_package(OpenSSL REQUIRED)
-find_package(Glog REQUIRED)
+find_package(Glog MODULE REQUIRED)
 find_package(DoubleConversion REQUIRED)
 find_package(Threads REQUIRED)
 if (UNIX AND NOT APPLE)
